doc/releases/changelog-dev.md

```diff
@@ -617,7 +617,7 @@ Users can specify the control wires as well as the values to control the operati
 
 This release contains contributions from (in alphabetical order):
 
-Kamal Ali,
+Kamal Mohamed Ali,
```

Comment:
The names should be in surname's alphabetical order

Reply:
Hey Albert, Ali is my surname, Mohamed is my middle name, so I believe this obeys alphabetical order